include/ola/Clock.h

```diff
@@ -257,5 +266,33 @@ class MockClock: public Clock {
  private:
   TimeInterval m_offset;
 };
+
+enum TimerGranularity { UNKNOWN, GOOD, BAD };
+
+/**
+ * @brief The Sleep class implements usleep with some granualtiry detection.
+ */
+class Sleep {
```

Comment:
I think this should probably be called GranularSleep or something should we ever have to do something like wrap/override usleep or similar.

Reply:
The whole point of this class is to do usleep without the various test systems screaming at us that usleep shouldn't be used.